src/models/Settings.js

```diff
@@ -52,6 +52,7 @@ function (Okta, Errors, BrowserFeatures) {
       'features.rememberMe': ['boolean', true, true],
       'features.rememberDevice': ['boolean', true, true],
       'features.rememberDeviceAlways': ['boolean', true, false],
+      'features.autoPush': ['boolean', true, false],
```

Comment:
the default value of autoPush in model is false, it shouldn't be `undefined`, because if it's `undefined`, you can't submit the form. Unless there is a code, say `parse` after `sync` set it to `undefined`.